### PR TITLE
Feat: Landing page

### PR DESCRIPTION
**Pull request by anamkaa**
Created Landing Page 

- Navbar 
- Hero Section 
- Category Section

Screenshot (375)

### CODE REVIEW

File: src/stylesheets/landingPage.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  align-items: center;
+  flex: 50%;
+}
+
+.ff-hero-container-text {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  align-items: flex-start;
+  color: white;
```
Comment (devdeadviz) [inline src/stylesheets/landingPage.css]: You can create a CSS variable for this color and utilize it wherever you want.

**Review by devdeadviz — APPROVED** [review]
Everything looks good to me.

**Review by MaruthiPothuganti — APPROVED** [review]
Everything looks good. PR looks good to merge.

File: src/stylesheets/landingPage.css (hunk 1 above)
Comment (anamkaa) [reply src/stylesheets/landingPage.css]: done

**Pull request merged**